src/seed.js

```diff
@@ -0,0 +1,486 @@
+export function seedDatabase(firebase) {
```

Comment:
Thx. Next time add a comment on the actual line of code so devs can go straight to the source.

Reply:
When you go to `Files Changed` tab. You can select the line that you want to add a comment too.